### ISSUE TITLE
qt: Does not work well with HiDPI displays on Linux

### ISSUE THREAD

**Issue by sdroege** [sdroege/gst-player#124]
@amorenoc see attached screenshot. How does it work on OSX with Retina displays?

It looks like parts of the UI are scaled properly, other parts not.
![qt](https://cloud.githubusercontent.com/assets/301846/10709716/3b4765de-7a41-11e5-96eb-9f76ccf4c4b6.jpg)

**Comment by amorenoc** [comment]
@sdroege retina displays looks fine, weird that in a HiDPI screen it's upscaled (since I am using device independent sizes for the icons).

**Comment by sdroege** [comment]
Well, the icons are upscaled and correct and so is the text (0:00). What is wrong is the scale of the window and the toolbar. They would have to be scaled by two too, then everything would look correct :)

Do you specify things in device pixels somewhere and could specify it in "visual pixels" instead?

**Comment by amorenoc** [comment]
@sdroege yes, I am specifying fonts in points and other sizes in pixels, so I will have to fix that :) 
But still puzzles me that I am not seeing this issue since I use a MacBook Pro with Retina display.

**Comment by amorenoc** [comment]
this link explains why we are seeing different results: http://blog.qt.io/blog/2013/04/25/retina-display-support-for-mac-os-ios-and-x11/
